### DISCUSSION TITLE
Deploy en azure con app service

### DISCUSSION THREAD

**Discussion by urielnat** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#539] [Q&A]
Buenas tardes, ¿Alguien pudo hacer el deploy de la práctica de vue.js usando **App service**?
hice los dos tipos de deploys, el primero como web estática el cual lo hizo sin problemas y en cuestión de segundos. Sin embargo como práctica adicional  decidí  intentar con app service, las actions dieron success pero aun así no actualiza la pagina que microsoft pone por defecto desde ya más de una hora
![image](https://user-images.githubusercontent.com/32392117/160258916-ddcb0092-e22c-4ceb-92e6-b56c78c83648.png) ¿Alguien si pudo realizarlo?
**Nota:** se que con webs estáticas de azure funciona y es más rápido

**Comment by doguedogue** [comment]
La verdad no, ya vi como le hizo el MC arriba, abajo y apretó todos lo botones, despues de varias actualizaciones del branch ya le levantó. 

No se que la hace tardar tanto, pero eventualmente te va a cargar. Igual y hay que revisar el manual antes.

Yo probé con github pages y no se si hay que cambiarle algo al contexto de la aplicación pero no me la cargó y tarda un poco también. Está más fácil con netlify ó heroku ó las azure estáticas (esas si jalan rápido)

![image](https://user-images.githubusercontent.com/23409026/160265695-a76583a5-6b52-4240-9700-69d4fac33c7d.png)
